Src/BootCamp.Chapter/MostCommonLetterFinder.cs

```diff
@@ -9,81 +9,40 @@ public static char Find(string sentence)
         {
             if (string.IsNullOrWhiteSpace(sentence))
             {
-                throw new ArgumentNullException();
+                throw new ArgumentNullException("The sentence you gave was empty.");
```

Comment:
This is fine, homeork normally you just pass a name of parameter using ``nameof(sentence)``